components/Faq.tsx

```diff
@@ -0,0 +1,28 @@
+"useClient";
```

Comment:
use client

Reply:
Hey @komal-agarwal5 yes it's working now, please do this small change and we are gtg

Reply:
Done